docs/APT_USER_DOCUMENTATION.md

```diff
@@ -0,0 +1,45 @@
+<!--
+
+    Sonatype Nexus (TM) Open Source Version
+    Copyright (c) 2018-present Sonatype, Inc.
+    All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+
+    This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+    which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+
+    Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+    of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+    Eclipse Foundation. All other trademarks are the property of their respective owners.
+
+-->
+## APT Repositories
+
+### Introduction
+
+[apt-get](https://help.ubuntu.com/community/AptGet/Howto) is a package management format used for Debian packages. 
+
+This allows the repository manager to take advantage of the packages in the official apt repositories and other
+public apt repositories without incurring repeated downloads of packages.
+
+### Proxying APT Repositories
+
+You can set up an apt proxy repository to access a remote repository location.
+
+To proxy a apt repository, you simply create a new 'apt (proxy)' as documented in 
+[Repository Management](https://help.sonatype.com/display/NXRM3/Configuration#Configuration-RepositoryManagement) in
+details. Minimal configuration steps are:
+
+- Define 'Name'
+- Define URL for 'Remote storage' e.g. []()
+- Select a 'Blob store' for 'Storage'
+
+### Configuring AptGet
+
+To configure apt-get to use Nexus Repository as a Proxy for remote apt sites 
+
+TBD
```

Comment:
For testing we do the following

1. Make a backup of /etc/apt/sources.list (cp /etc/apt/sources.list /etc/apt/sources.backup.list)
2. Remove sources.list (rm /etc/apt/sources.list)
3. Create a new empty sources.list (touch /etc/apt/sources.list)
4. Edit the sources.list and add the following lines (changing the distribution xenial to be whatever your distribution is)

deb http://nxrm-ip:8081/repository/apt-proxy/ xenial main restricted
deb-src http://nxrm-ip:8081/repository/apt-proxy/ xenial main restricted